assembl/static2/js/app/reducers/adminReducer/index.js

```diff
@@ -63,10 +49,47 @@ export const thematicsById: ThematicsByIdReducer = (state = Map(), action) => {
       titleEntries: List(),
       video: null
     });
-    return state.set(action.id, emptyThematic.set('id', action.id));
+    const order = state.size + 1.0;
```

Comment:
Why has order to be a float ?

Reply:
This is a float in the backend, but we use it only as a integer at the end in the fronted.